test/app_signals/resources/metrics/client_producer.json

```diff
@@ -34,8 +34,8 @@
           "metrics": [
             {
               "name": "Error",
-              "unit": "Milliseconds",
-              "sum": {
+              "unit": "",
+              "exponential_histogram": {
```

Comment:
Whats the difference between exponential histogram and sum

Reply:
exponential_histogram is the data type that app signal uses for its metrics. We had the wrong data type which is why the metrics were generating invalid metrics